Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -251,6 +251,14 @@ class TelemetryWrapper {
         GleanMetrics.InstalledMozillaProducts.klar.set(UIApplication.shared.canOpenURL(URL(string: "firefox-klar://")!))
         // Device Authentication
         GleanMetrics.Device.authentication.set(AppAuthenticator.canAuthenticateDeviceOwner())
+
+        // Wallpapers
+        let currentWallpaper = WallpaperManager().currentWallpaper
+
+        if case .themed = currentWallpaper.type {
+            GleanMetrics.WallpaperAnalytics.themedWallpaper["wallpaperName-"+currentWallpaper.name].add()
```

Comment:
As per our discussion, this doesn't match up with the actual label. We should be sending the wallpaper name in an extra (to account for future wallpapers that don't ship with the app). Looking at `metrics.yaml` this seems to imply this should be an event.

@dnarcese Do we want statisticts on which wallpapers are being used, or whether or not wallpapers are being used? One can be a counter, one cannot, I think.